src/RenderWebGL.js

```diff
@@ -1096,6 +1096,14 @@ class RenderWebGL extends EventEmitter {
      * @private
      */
     _drawThese (drawables, drawMode, projection, opts = {}) {
+
+        const near = function (a, b, relativeTolerance = 0.01) {
+            const absA = Math.abs(a);
+            const absB = Math.abs(b);
+            const error = Math.abs(a - b) / Math.max(absA, absB);
+            return error < relativeTolerance;
+        };
+
```

Comment:
I wrote this function when I was attempting to compare dimensions instead of scaling factors... it might be overly complicated now but it does make it clear what I'm doing, so... opinions?